lib/transports/elasticsearch.js

```diff
@@ -121,6 +121,10 @@ class elasticsearch {
     aws4signer(esRequest, this.parent)
     this.baseRequest(esRequest, (err, response) => {
       if (err) { return callback(err) }
+      if (response.statusCode !== 200) {
+        const errMsg = `An error occurred: [${response.statusCode}] ${response.body}`
```

Comment:
Using no semi-colon and declared a new variable to comply with repo's coding guidelines. `npm run test` report:
![image](https://user-images.githubusercontent.com/18154526/98680673-a5767f00-2359-11eb-8c9e-2b863c4a7ec7.png)